frontend/routes/home/index.tsx

```diff
@@ -1,8 +1,10 @@
 import { Navigate } from 'react-router-dom'
 import { useGetRedirectPath } from '../../hooks/redirect-path'
+import { useSentry } from '../../hooks/sentry'
 
 export const Home = () => {
   const { loading, error, path } = useGetRedirectPath()
+  useSentry()
```

Comment:
**question (non-blocking)**:  I feel like this means that telemetry defaults to on? Is that alright?

Reply:
Nope, use sentry checks the settings and only enables if the setting has been saved by the user

Reply:
https://github.com/vegaprotocol/vegawallet-browser/pull/317/files#diff-2a5e93a24268b8525fdd0faa490d14140966605c64c4ff868c3daea3c3e5cc90R29